week-1/G-strings-methods/exercise2.js

```diff
@@ -1,3 +1,8 @@
-var name = " Daniel  ";
+var name = " Daniel  ".trim();
+
+var message =
+  " My name is" + name + "and my name has " + name.length + " letters";
+
+console.log(name.trim());
```

Comment:
Do you think using `.trim()` a second time here is necessary? What happens if we remove it? Can you explain why?

Reply:
That's great that you saw this. I used the method twice because I was trying to see if it would work the same way. But then I found out a better and clearer way to solve it by updating the variable instead of defining with the method inside. What do you think? Thank you!

Reply:
Nice fix 👍